src/server.c

```diff
@@ -58,7 +58,6 @@ void print_help() {
                     "    --uid, -u               User id to run with\n"
                     "    --gid, -g               Group id to run with\n"
                     "    --signal, -s            Signal to send to the command when exit it (default: SIGHUP)\n"
-                    "    --signal-list           Print a list of supported signals\n"
```

Comment:
Thanks, I agree that we should keep ttyd simple, let's remove the `--signal-list `.

The ssl/uid/gid functionalities are exposed from libwebsockets, just for convenience, I won't remove them right now, since they had been added and are not required options.